EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+
+3. If desired, check if the condition has been really marked for a later data mapping. To do this, open the message again.
+
+4. Click the *Data sources* tab and the related data source.  
+   - The *Map Attributes* wizard window is displayed.
+   - The *EDI Export Condition FTX* has been added to the attributes to be mapped.
+
+    ![Condition for mapping](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefMappingCondition.png)
+
+5. Finalize the *Map attributes* wizard by clicking the [CONTINUE] buttons until you can click the [FINALIZE] button.
+  
+6. Click the [SAVE] button to leave the header definition.   
+   The *Exporter definitions* view is displayed.
+
+7. Continue specifying the header file segments until you have specified all fields that your business partner require. By doing this, do not specify the segments that are relevant for the line item export. For the INVOIC, it is the SG26 or SG27 segment that contains the S_LIN specification for the line item export. For detailed information, see [Prepare line item export](#prepare-line-item-export) below.
+
+
+
+## Prepare line item export
+
+In order to be able to export the individual line items of an invoice with specific data sources later, the relevant line item group of an EDIFACT message must be specified to be exported as a whole. You have to do this here in the header definition file settings.  
+After you have prepared the line item export, you can define the definition for the line items, see [Manage definitions](./01_ManageDefinitions.md).
+
+This procedure shows how you must prepare the line item section of the message, so that the *DataHub Exporter* can add all line items to an invoice message later.
+
+#### Prerequisites
+
+- You have expanded the message structure, see [Get header file segments](#get-header-file-segments).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Select message definition > File settings tab > Expand message structure*
+
+![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefDates.png "[Message structure]")
+
+
+1. Check the EDIFACT message structure, in which group the LIN segment is available. In newer versions of the INVOIC, the LIN segment is included in group 27, in older versions it is in group 26.
+
+2. Scroll down in the message structure until you have found the relevant group, *G_INVOIC_07A_SG26* or *G_INVOIC_13B_SG27*.
+
+3. Click the ![Add XML attribute](../../Assets/Icons/Plus08.png "[Add XML attribute]") (Add XML attribute) button to the right of the *G_INVOIC_07A_SG26* resp. *G_INVOIC_13B_SG27* row. 
+    - The segment is highlighted with a small black border. You cannot expand it.      
+    - Another *G_INVOIC_07A_SG26* or *G_INVOIC_13B_SG27* row has been added. You can ignore it.      
+    - The EDI Export SG26/SG27 *Destination attribute* has been marked for a later data mapping.
+
+4. Click the [SAVE] button.    
+   The *Exporter definitions* view is displayed.
+
```

Comment:
Kann man hier nicht dir Prozedur abschließen und ggf. weiter zum Mappen in einer separaten Prozedur?